indra/newview/llappviewer.cpp

```diff
@@ -5200,6 +5200,11 @@ void LLAppViewer::updateNameLookupUrl(const LLViewerRegion * regionp)
     }
 }
 
+void LLAppViewer::postToMainThread(const LL::WorkQueue::Work& work)
```

Comment:
Ah - I was looking for this in existing code when I saw your comment referencing it.

But maybe `postToMainCoro()` ?

Reply:
co-pilot hallucinated the function and named it for me :)

Reply:
You know what, why not make the `"mainloop"` `WorkQueue` a member of `LLAppViewer`? Through the magic of `LLInstanceTracker::getInstance()`, it could still be found by its string name, but I like the idea of `LLAppViewer` actually owning it.